### ISSUE TITLE
Generate constants for JFR event type names and attributes

### ISSUE THREAD

**Issue by gunnarmorling** [moditect/jfrunit#65]
Allowing for a safe access to these literals in testing code.

**Comment by Croway** [comment]
yes, you are right, it's much better in this way, and I don't think it will be to hard to implement, in order to generate classes I created a simple code generator with freemarker that gets all the events from https://bestsolution-at.github.io/jfr-doc/openjdk-17.json (luckly there is a json api :smile: ), once I'll do the changes I'll share the code generator.

**Comment by gunnarmorling** [comment]
I'd definitely prefer option a), making this a multi-module project. This will allow for quicker turnaround times after changes to the generator, and I think it just makes more sense to maintain everything in one place.
